libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/dialogs/ConfirmationDialog.kt

```diff
@@ -37,11 +37,11 @@ import io.element.android.libraries.ui.strings.R as StringR
 
 @Composable
 fun ConfirmationDialog(
-    title: String,
     content: String,
     onSubmitClicked: () -> Unit,
     onDismiss: () -> Unit,
     modifier: Modifier = Modifier,
+    title: String? = null,
```

Comment:
Is is good practice to have dialog without title? Maybe the design team could confirm that.

Reply:
I think every single dialog in the room details screen has no title. @callumu can you confirm that's what we want?